packages/react-components/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,422 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import { Button } from '../Button';
+import { ITooltipProps, Tooltip as TooltipComponent } from './Tooltip';
+import {
+  DropDown,
+  Smiles,
+  CannedResponse,
+  Attachment,
+} from '@livechat/design-system-icons/react/material';
+
+import { Icon } from '../Icon';
+
+import './Tooltip.stories.css';
+import { Simple } from './Simple';
+import { Info as TooltipInfoComponent } from './Info';
+import { Interactive as TooltipInteractiveComponent } from './Interactive';
+import { UserGuide as TooltipUserGuideComponent } from './UserGuide';
+import { UserGuideStep } from './UserGuideStep';
+import beutifulImage from './placeholder.png';
+
+const tooltipPlacements = [
+  'bottom',
+  'bottom-end',
+  'bottom-start',
+  'left',
+  'left-end',
+  'left-start',
+  'right',
+  'right-end',
+  'right-start',
+  'top',
+  'top-end',
+  'top-start',
+];
+
+const isVisibleOptions = [true, false, undefined];
+
+const tooltipThemes = ['invert', 'important', 'default'];
+
+export const Tooltip = (args: any): React.ReactElement => (
+  <div
+    style={{
+      height: '100vh',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    }}
+  >
+    <span className={'tooltip-icon-wrap'}>
+      <TooltipSimpleExample {...args}></TooltipSimpleExample>
+    </span>
+    <span className={'tooltip-icon-wrap'}>
+      <Icon source={CannedResponse}></Icon>
+    </span>
+    <span className={'tooltip-icon-wrap'}>
+      <Icon source={Attachment}></Icon>
+    </span>
+  </div>
+);
+
+Tooltip.args = {
+  placement: 'bottom',
+  isVisible: true,
+  theme: 'default',
+  triggerOnClick: false,
+  arrowOffsetY: 0,
+  arrowOffsetX: 0,
+  offsetMainAxis: 8,
+  withFadeAnimation: true,
+  transitionDuration: 200,
+  transitionDelay: 0,
+  hoverOutDelayTimeout: 100,
+};
+
+export const TooltipInfo = (args: any) => (
+  <div
+    style={{
+      height: '100vh',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    }}
+  >
+    <TooltipInfoExample {...args}></TooltipInfoExample>
+  </div>
+);
+
+TooltipInfo.args = {
+  placement: 'bottom',
+  isVisible: true,
```

Comment:
On the TooltipInfo story it would be great to use the button which is already there and showcase the simple managed tooltip case so button click => tooltip is shown, close button or esc => tooltip is closed
```suggestion
```